mubench.reviewsite/src/Model/Review.php

```diff
@@ -45,6 +45,21 @@ public function getHitViolationTypes($rank){
         return $this->getReviewFinding($rank)['violation_types'];
     }
 
+    public function hasPositiveHit()
+    {
+        return $this->getDecision() == Decision::YES;
+    }
+
+    public function getMinimalPositiveHitRank()
```

Comment:
Please rename to `getLowestHitRank`. Please also throw an exception, when there is no hit.